app/build.gradle

```diff
@@ -35,24 +35,17 @@ android {
 }
 dependencies {
     // Build plugins
-    // TODO: Figure out why removing this causes a Dagger error.
-    provided 'com.github.frankiesardo:auto-parcel-processor:0.2' // Automatic Parcelables
     provided 'com.squareup.dagger:dagger-compiler:1.2.2' // Dependency injection
-    // provided 'com.stanfy.spoon:spoon-gradle-plugin:0.14.1' // Test multiplexing/selection
 
     // Local .jar dependencies
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
     // Local project dependencies
     compile project(':third_party:odkcollect')
-    compile project(':third_party:AutoFitTextView')
```

Comment:
@capnfabs How dumb am I? It's the Readme for the AutoFitTextView project. ahahahahaha.
Located in:
/third_party/AutoFitTextView/README.md
Sorry!